src/ui/sub_views/details_pane.rs

```diff
@@ -114,6 +125,16 @@ impl DetailsPane
             offset: 0,
         }
         .draw(ctx, f, req_resp_chunks[1]);
+
+        // The right side view is split vertically only if the client included its process id and thread id in the request
+        // enabling the callstack capture.
+        if vertical_chunks.len() > 1 {
+            CallstackView {
+                request: request.request_data.uuid,
+                offset: 0,
+            }
+            .draw(ctx, f, vertical_chunks[1]);
+        }
```

Comment:
Going forward, it might make sense to give all rects a name.

As in, after `split`, acquire references to individual rects:

```rust
let (upper_chunk, maybe_callstack_chunk) = if ... {
  let split = Layout::default()...split(block.inner(c))
  (split[0], Some(split[1]))
} else {
  (block.inner(c), None)
};

let (request_chunk, response_chunk) = {
    let split = Layout::default()...split(upper_chunk);
    (split[0], split[1])
};
```

That would allow us to use named chunks here in the draw-calls instead of (somewhat) arbitrary `[0]` and `[1]` indexes.

Especially I'd like the option of replacing the `if` here with:
```rust
if let Some(callstack_chunk) = maybe_callstack_chunk {
    /* draw */
}
```